projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -114,12 +96,14 @@ export function checkQuoteStatusInQuoteList(status: string) {
     checkQuoteStatusInQuoteList.name
   );
   cy.get('@quoteId').then((quoteId) => {
-    cy.get(quoteListSelector + rowSelector)
-      .contains(codeCellSelector, `${quoteId}`)
-      .parent()
-      .within(() => {
-        cy.get(statusCellSelector).contains(statusCellSelector, status);
-      });
+    cy.get(quoteListComponentSelector).within(() => {
+      cy.get('tr')
+        .contains('td.cx-code', `${quoteId}`)
+        .parent()
+        .within(() => {
+          cy.get('td.cx-status').contains('td.cx-status', status);
```

Comment:
same here do we need td.cx-status or is .cx-status already unique?

Reply:
there are few more occurrences of this pattern, were the td-tag can be omitted in the selector as well.

Reply:
you are right! I shortened all the identifiers from td.xxxx to .xxxx